### PR TITLE
[Keymap] Add new 42 keymap for the cantor keyboard

### PR DESCRIPTION
**Pull request by konova8**
## Description

Added new keymap for the cantor keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cantor/keymaps/Cobbleston/config.h [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
```
Comment (zvecr) [inline keyboards/cantor/keymaps/Cobbleston/config.h:46]:
```suggestion
```

File: keyboards/cantor/keymaps/Cobbleston/keymap-markstos-like.c [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/cantor/keymaps/Cobbleston/keymap-markstos-like.c:1]: File should be deleted

File: keyboards/cantor/keymaps/Cobbleston/keymap-miryoku-like.c [hunk 3]
```diff
@@ -0,0 +1,71 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/cantor/keymaps/Cobbleston/keymap-miryoku-like.c:1]: File should be deleted

File: keyboards/cantor/keymaps/Cobbleston/keymap.c [hunk 4]
```diff
@@ -0,0 +1,78 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/cantor/keymaps/Cobbleston/keymap.c:1]: Needs licence header

**Review by zvecr — CHANGES_REQUESTED** [review]
* "Cobbleston" is an invalid keymap name due to the capital letter, folder needs to be renamed
* pdf files are not accepted to this repo, they need to be deleted

**Comment by Cobbleston** [comment]
Now it should be ok I think, do I need to check all 14 checks before being able to merge in the mail repo?

File: keyboards/cantor/keymaps/konova8/config.h [hunk 5]
```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Diego Palacios (@diepala)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/cantor/keymaps/konova8/config.h:6]:
```suggestion
```

**Review by drashna — APPROVED** [review]

**Comment by konova8** [comment]
Sorry to bother, is there still something I have to do before being able to merge?
Thank you!

File: keyboards/cantor/keymaps/konova8/config.h [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
```
Comment (fauxpark) [inline keyboards/cantor/keymaps/konova8/config.h:45]:
```suggestion
```

**Review by fauxpark — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Wrong button

**Comment by zvecr** [comment]
Unfortunately since this PRs creation, QMK has [changed its stance](https://docs.qmk.fm/#/ChangeLog/20230827?id=notable-changes) on accepting user keymaps to the repository.

> qmk/qmk_firmware is no longer accepting PRs for keymaps other than for manufacturer-supported keymaps. User keymap workflow has been documented [here](https://docs.qmk.fm/#/newbs) for several years. This change is to progressively reduce the maintenance burden on the project, and to allow us to focus on the core features of QMK.

In the near future we will be providing a more user focused workflow for keymaps, but till then the suggestion is to keep changes in your own fork.

While we appreciate the time that was taken on this PR, hopefully you understand we are closing it, so that QMK can improve in the long term.

**Pull request closed without merging**